### ISSUE TITLE
Optimizer.predict()

### ISSUE THREAD

**Issue by SRFU-NN** [novonordisk-research/ProcessOptimizer#219]
Add a Optimizer.predict method that handles all of the nastyness.

The code can be taken from Krigin believer in Optimizer.ask

**Comment by SRFU-NN** [comment]
Of the optional returns in model.predict (return_std, return_cov, return_mean_grad, return_std_grad), only retrun_std will be implemented. The gradients change with the transformation of X, and the transformation is non-trivial, so I don't want to code it. The covariance matrix could make sense, but I am not sure that it would be useful.

On the other hand, I would like to always return std. Users should take it into account, so we might as well nudge them to do so. It is more code to run, so it will take longer, but I feel like that is OK.

**Comment by SRFU-NN** [comment]
I would prefer to change the format of the what is returned, compared to model.predict.

model.predict(X, return_std=True) returns a tuple, whihc ocntains a list of the means and a list of the std's.

I would like to return a list of named tuples, with each named tuple having a field per objective, and each of these fields containing a named tuple with the fields "mean" and "std". So the mean and std of each point are tightly coupled, as opposed to the situation in the return of model.predict, where there are only linked by having the same position in the two lists.

Edit: model.predict's behavior is inherited from [scikit-learn](https://scikit-learn.org/stable/modules/generated/sklearn.gaussian_process.GaussianProcessRegressor.html#sklearn.gaussian_process.GaussianProcessRegressor.predict), where it makes sense since most objects that have a predict function can only return one number, and no std. But we are in the opposite position: Everything (?) we make has the ability to also return a std, including our extra trees regressor. This is in opposition to scikit-learn's EctraTreesRegressor.

The solution is to call the new method "estimate".
